references/classification/presets.py

```diff
@@ -1,8 +1,22 @@
 import torch
-from torchvision.transforms import autoaugment, transforms
 from torchvision.transforms.functional import InterpolationMode
 
 
+def get_modules(use_v2):
+    # We need lazy import to avoid the V2 warning in case just V1 is used
+    import torchvision
+
+    if use_v2:
+        import torchvision.transforms.v2
+
+        return torchvision.transforms.v2, torchvision.transforms.v2
+    else:
+        import torchvision.transforms
+        import torchvision.transforms.autoaugment
+
+        return torchvision.transforms, torchvision.transforms.autoaugment
```

Comment:
Also falls into the cleanup category, but maybe we can do that here: v1 also exposes all AA transforms under `torchvision.transforms`

https://github.com/pytorch/vision/blob/23b0938f897f4003bb26c086115de94b9976cb9f/torchvision/transforms/__init__.py#L2

Meaning, there is no need for two modules here.